tensorflow_datasets/core/download/downloader.py

```diff
@@ -115,15 +115,15 @@ def download(self, url: str, destination_path: str):
 
   def _sync_kaggle_download(self, kaggle_url, destination_path):
     """Download with Kaggle API."""
-    kaggle_file = kaggle.KaggleFile.from_url(kaggle_url)
-    downloader = self.kaggle_downloader(kaggle_file.competition)
-    filepath = downloader.download_file(kaggle_file.filename, destination_path)
-
+    downloader = self.kaggle_downloader(kaggle_url)
```

Comment:
As the files aren't downloaded by the `_downloader` anymore, this `_sync_kaggle_download` could be removed too.

Reply:
FIxed.